memcached-operator/pkg/apis/cache/v1alpha1/memcached_types.go

```diff
@@ -26,7 +26,6 @@ type MemcachedStatus struct {
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 
 	// Nodes are the names of the memcached pods
-	// +kubebuilder:validation:UniqueItems=true
```

Comment:
Any reason for removing this?

Reply:
It will not work with the API validation and should not be added in the first place since was just a test to check it.